atomic_reactor/plugins/pre_resolve_remote_source.py

```diff
@@ -116,6 +117,7 @@ def set_worker_params(self, source_request, remote_source_url):
         }
         override_build_kwarg(self.workflow, 'remote_source_url', remote_source_url)
         override_build_kwarg(self.workflow, 'remote_source_build_args', build_args)
+        override_build_kwarg(self.workflow, 'remote_source_configs', remote_source_configs)
```

Comment:
What does this do?

Reply:
I sets the parameter to be forwarded to the worker builds. This is then set as a param to the download_remote_source plugin.

This is needed because while this plugin runs in the orchestrator build, the download_remote_sources plugin (which prepares the sources to be copied into the build image) runs in the worker builds.